primitives/src/pcs/univariate_kzg/mod.rs

```diff
@@ -322,16 +332,45 @@ impl<E: Pairing> UnivariatePCS for UnivariateKzgPCS<E> {
         num_points: usize,
         domain: &Radix2EvaluationDomain<Self::Evaluation>,
     ) -> Result<Vec<Self::Proof>, PCSError> {
-        let mut h_poly = Self::compute_h_poly_in_fk23(prover_param, &polynomial.coeffs)?;
-        let proofs: Vec<_> = h_poly
-            .batch_evaluate_rou(domain)?
-            .into_iter()
-            .take(num_points)
-            .map(|g| UnivariateKzgProof {
-                proof: g.into_affine(),
-            })
-            .collect();
-        Ok(proofs)
+        #[cfg(feature = "no-fk23")]
```

Comment:
personal preference: 
`no-fk23` sounds cryptic, I prefer `naive-kzg-multi-open` which is explicit and clear.

Reply:
done in c3373a0f47be9f1f4342f997076b8df56d9ba45a